lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -387,7 +388,7 @@ def add_stored_credentials_using_normalized_fields(post, options)
       def add_stored_credential_options(post, options = {})
         return unless options[:stored_credential]
 
-        post[:payment_type] = 'Recurring' if %w(recurring installment).include? options[:stored_credential][:reason_type]
+        post[:payment_type] = options[:stored_credential][:reason_type].capitalize if options[:stored_credential][:reason_type]
```

Comment:
I suggest removing the condition for the ampersand (&)

```suggestion
        post[:payment_type] = options[:stored_credential][:reason_type]&.capitalize
```

Reply:
added!